lib/networkx/graph.rb

```diff
@@ -325,5 +325,13 @@ def edge_subgraph(edges)
         "received #{edges.class.name} instead."
       end
     end
+
+    def is_multigraph
+      self.class.name.include?('Multi')
+    end
+
+    def is_directed
```

Comment:
Also, it'd be a more Rubyistic nomenclature to rename these methods with a `?` like `is_multigraph?` and `is_directed?`.

Reply:
I saw an offense related to these two. Will modify it shortly.